Assets/_Functionalities/ObjectInformation/Scripts/FeatureMapping.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections.Generic;
+using GeoJSON.Net.Feature;
+using Netherlands3D.SubObjects;
+using Netherlands3D.Twin.Layers;
+using UnityEngine;
+
+namespace Netherlands3D.Twin
+{
+    public class FeatureMapping : MonoBehaviour
+    {
+        public string FeatureID { get { return feature.Id; } }
```

Comment:
als er alleen een get is dan zou ik het schrijven als `FeatureID => feature.Id;` dat vind ik iets leesbaarder